docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  book-shop-db:
+    container_name: book-shop-db
+    image: mysql:8
+    restart: unless-stopped
+    env_file: ./.env
+    ports:
+      - $MYSQLDB_LOCAL_PORT:$MYSQLDB_DOCKER_PORT
+    environment:
+      - MYSQL_ROOT_PASSWORD=$MYSQLDB_ROOT_PASSWORD
+      - MYSQL_DATABASE=$MYSQLDB_DATABASE
+  book-app:
+    build: .
+    container_name: book-app
+    image: book-app
+    restart: on-failure
+    depends_on:
+      - book-shop-db
+    env_file: ./.env
+    ports:
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url" : "jdbc:mysql://book-shop-db:$MYSQLDB_DOCKER_PORT/$MYSQLDB_DATABASE?serverTimezone=UTC",
+        "spring.datasource.username" : "$MYSQLDB_USER",
+        "spring.datasource.password" : "$MYSQLDB_ROOT_PASSWORD",
+        "spring.datasource.driverClassName" : "com.mysql.cj.jdbc.Driver"
```

Comment:
You don't really need to explicitly specify driver. It will be automatically got from your application.yml/properties before the liquibase migration starts apply itself to db.